<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTerminal</class>
 <widget class="QWidget" name="CTerminal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="CTextEdit" name="textEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="sendHorizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="CLineEdit" name="sendLineEdit" native="true"/>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CCheckbox" name="logCheckBox">
       <property name="text">
        <string>Log</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CLineEdit</class>
   <extends>QWidget</extends>
   <header>CSdk/CLineEdit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CTextEdit</class>
   <extends>QTextEdit</extends>
   <header>CSdk/CTextEdit.h</header>
  </customwidget>
  <customwidget>
   <class>CCheckbox</class>
   <extends>QLabel</extends>
   <header>CSdk/CCheckbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
